WangscapeTest/TestModuleCodec.cpp

```diff
@@ -0,0 +1,95 @@
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <algorithm>
+#include <string>
+
+#include <noise/module/codecs/ModuleCodecs.h>
+
+class TestModuleCodec : public ::testing::Test
+{
+protected:
+    void remove_whitespace(std::string& s)
+    {
+        s.erase(std::remove_if(s.begin(),
+                               s.end(),
+                               [](char c) { return std::isspace(c, std::locale()); }),
+                s.end());
+    }
+};
```

Comment:
It looks the same as in `WangscapeTest/TestEncodedModuleGroup.cpp`. Maybe just move it to a separate file 😃.